[DOCS] Reduce execution terminology in actions readme (#136342)
elastic/kibana: x-pack/plugins/actions/README.md

**Action Type**: A programatically defined integration with another service, with an expected set of configuration and parameters properties, typically defined with a schema. Plugins can add new
action types.

**Action**: A configuration object associated with an action type, that is ready to run. The configuration is persisted via Saved Objects, and some/none/all of the configuration properties can be stored encrypted.

## Usage

1. Develop and register an action type (see [Action types -> Example](#example)).
2. Create an action by using the [RESTful API](#restful-api).

> 3. Use alerts to generate actions or create them manually (see [Firing actions](#firing-actions)).

Reviewer: create seems wrong as actions aren't created.
They're either attached to a rule which will then schedule them when an alert is created, or they're triggered manually.

```suggestion
3. Use alerting rules to generate actions or trigger them manually (see [Firing actions](#firing-actions)).
```

#### **allowedHosts** configuration

> - You can use the string "*" in the **allowedHosts** configuration in place of a specific hostname to enable Kibana to target any URL, but keep in mind the potential to use such a feature to create [SSRF](https://owasp.org/www-community/attacks/Server_Side_Request_Forgery) attacks from your server.

Reviewer: This is warning about a bad actor using actions to attack another server - create feels wrong, as an attack is performed or launched 🤔

Author: Thanks, I changed it to "launch"

Reviewer: 🚀

> - You can use the string "*" in the **allowedHosts** configuration in place of a specific hostname to enable Kibana to target any URL, but keep in mind the potential to use such a feature to create [SSRF](https://owasp.org/www-community/attacks/Server_Side_Request_Forgery) attacks from your server.

Author: ```suggestion
- You can use the string "*" in the **allowedHosts** configuration in place of a specific hostname to enable Kibana to target any URL, but keep in mind the potential to use such a feature to lauch [SSRF](https://owasp.org/www-community/attacks/Server_Side_Request_Forgery) attacks from your server.
```

### Executor

This is the primary function for an action type. Whenever the action needs to run, this function will perform the action. It receives a variety of parameters. The following table describes the properties that the executor receives.

**executor(options)**

| Property                                | Description                                                                                                                                                                                                                                                                                                                                                                                 |
| --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | actionId                                | The action saved object id that the action type is generating.                                                                                                                                                                                                                                                                                                                           |

Reviewer: ```suggestion
| actionId                                | The action saved object id that the connector is generating.                                                                                                                                                                                                                                                                                                                           |
```
